code/src/index.css

```diff
@@ -11,3 +18,40 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.todo-list-card {
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+  background: linear-gradient(
+    189deg,
+    rgba(166, 228, 255, 1) 0%,
+    rgba(125, 113, 195, 1) 72%
+  );
+  padding: 30px;
+  width: auto;
```

Comment:
It is not a good practice to do that. You're completely losing control over your container.
If you still want to use it as that you need to think about making your inputs stretch ass well, or move your content to the center.